backend/kernelCI_app/views/treeDetailsSlowView.py

```diff
@@ -349,11 +358,10 @@ def __processNonBootsTest(self, currentRowData):
             self.testPlatformsWithErrors.add(testPlatform)
             self.testFailReasons[testError] = self.testFailReasons.get(testError, 0) + 1
 
-        if testEnvironmentCompatible is not None:
-            if testStatus is None:
-                self.testEnvironmentCompatible[testEnvironmentCompatible]["NULL"] += 1
-            else:
-                self.testEnvironmentCompatible[testEnvironmentCompatible][testStatus] += 1
+        if testStatus is None:
+            self.testEnvironmentCompatible[testEnvironmentCompatible]["NULL"] += 1
+        else:
```

Comment:
I think `testStatus` will never be `None` here because it is treated in `__getCurrentRowData`